### PR TITLE
[Merged by Bors] - feat(RelSeries): eraseLast_last_rel_last

### PR DESCRIPTION
**Pull request by nomeata**
from the Carleson project

### CODE REVIEW

File: Mathlib/Order/RelSeries.lean [hunk 1]
```diff
@@ -477,6 +477,14 @@ def eraseLast (p : RelSeries r) : RelSeries r where
 
 @[simp] lemma last_eraseLast (p : RelSeries r) :
     p.eraseLast.last = p ⟨p.length.pred, Nat.lt_succ_iff.2 (Nat.pred_le _)⟩ := rfl
+
+/-- In a non-trivial series `p`, the last element of `p.eraseLast` is related to `p.Last` -/
```
Comment (Ruben-VandeVelde) [inline Mathlib/Order/RelSeries.lean:481]: Did you mean:
```suggestion
/-- In a non-trivial series `p`, the last element of `p.eraseLast` is related to `p.last` -/
```

File: Mathlib/Order/RelSeries.lean [hunk 2]
```diff
@@ -477,6 +477,14 @@ def eraseLast (p : RelSeries r) : RelSeries r where
...
+lemma eraseLast_last_rel_last (p : RelSeries r) (h : 0 < p.length) :
```
Comment (Ruben-VandeVelde) [inline Mathlib/Order/RelSeries.lean:482]: I think we adopted the policy that hypotheses should be of the form x \ne 0, not 0 < x

File: Mathlib/Order/RelSeries.lean (hunk 1 above)
Comment (nomeata) [reply Mathlib/Order/RelSeries.lean:481]: Yes, not sure how that got capitalized.

File: Mathlib/Order/RelSeries.lean (hunk 2 above)
Comment (nomeata) [reply Mathlib/Order/RelSeries.lean:482]: Thanks! Will update

Comment (nomeata) [reply Mathlib/Order/RelSeries.lean:482]: Thanks to `omega`, no changes needed in the proof nor its use in my project. Nice.
```suggestion
lemma eraseLast_last_rel_last (p : RelSeries r) (h : p.length ≠ 0) :
```

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

**Pull request closed without merging**